panacea/grpc_client.go

```diff
@@ -124,13 +126,13 @@ func (c *GrpcClient) GetDeal(id string) (datadealtypes.Deal, error) {
 }
 
 // GetRegisteredOracle gets registered oracle
-func (c *GrpcClient) GetRegisteredOracle(address string) (*datapooltypes.Oracle, error) {
-	client := datapooltypes.NewQueryClient(c.conn)
+func (c *GrpcClient) GetRegisteredOracle(address string) (*oracletypes.Oracle, error) {
```

Comment:
I think this method is not requirement.
Because already in `x/oracle` module.
We should consider removing this function.